scipy/stats/tests/test_stats.py

```diff
@@ -1771,6 +1771,30 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[0], z0_expected)
         assert_array_almost_equal(z[1], z1_expected)
 
+    def test_zscore_nan_propagate(self):
+        # Test nan_policy set to 'propogate'
+        x = np.array([1, 2, np.nan, 4, 5])
+
+        z = stats.zscore(x, nan_policy='propagate')
+
+        expected = np.array([np.nan, np.nan, np.nan, np.nan, np.nan])
```

Comment:
Could just do `assert all(np.isnan(z))`.

Reply:
Thanks for this idea. I will do this. I'm going to update a PR with all these changes shortly.